shell/initialize/client.js

```diff
@@ -207,16 +123,6 @@ async function render(to, from, next) {
     return next();
   }
 
-  if (to === from) {
-    _lastPaths = [];
-  } else {
-    const fromMatches = [];
-
-    _lastPaths = getMatchedComponents(from, fromMatches).map((Component, i) => {
-      return compile(from.matched[fromMatches[i]].path)(from.params);
-    });
-  }
```

Comment:
After removing other code _lastPaths was never read it was only written to.

The risk to removing code in a scenario like this is that a large portion of our code has side effects even in getters. I did look at getMatchedComponents and compile and neither appear to have side effects.